### PR TITLE
Sets cifmw_openshift_setup_kubeconfig for rdo crc-base jobs

### PR DESCRIPTION
**Pull request by viroel**
This PR has:
- [ ] Appropriate testing (molecule, python unit tests)
- [ ] Appropriate documentation (README in the role, main README is up-to-date)

### CODE REVIEW

File: zuul.d/edpm.yaml [hunk 1]
```diff
@@ -35,6 +35,8 @@
       network_isolation: true
       crc_attach_default_interface: true
       bmo_setup: false
+      # This should be set here since is associated with parent job (base-crc) and not the scenario
+      cifmw_openshift_setup_kubeconfig: "/home/zuul-worker/.crc/machines/crc/kubeconfig"
```
Comment (pablintino) [inline zuul.d/edpm.yaml:39]: The variable we created for this purpose is cifmw_kubeconfig

Comment (viroel) [reply zuul.d/edpm.yaml:39]: sorry, let me update that

Comment (pablintino) [reply zuul.d/edpm.yaml:39]: Don't worry, this fault went through the CI, I think we should enable the edpm job in case we touch this role too

**Pull request merged**